blocks/test/full-content.js

```diff
@@ -124,10 +123,7 @@ describe( 'full post content fixture', () => {
 			const parserOutputExpected = JSON.parse( parserOutputExpectedString );
 			expect(
 				parserOutputActual
-			).to.eql(
-				parserOutputExpected,
-				format( 'File \'%s.parsed.json\' does not match expected value', f )
-			);
+			).toEqual( parserOutputExpected );
```

Comment:
This is what I see when I make JSON.parse throw an error:

> ● full post content fixture › core__html
> 
>     SyntaxError: Unexpected token x in JSON at position 0
>         at Object.parse (native)
>       
>       at Object.<anonymous> (blocks/test/full-content.js:123:36)
>       at Promise.resolve.then.el (node_modules/p-map/index.js:42:16)
>       at process._tickCallback (internal/process/next_tick.js:109:7)

It seems okey from my perspective. We can also code our own matcher which would print error using [expect.extend](https://facebook.github.io/jest/docs/en/expect.html#expectextendmatchers).